src/backend/core/api/serializers.py

```diff
@@ -532,3 +533,26 @@ class MoveItemSerializer(serializers.Serializer):
     """
 
     target_item_id = serializers.UUIDField(required=True)
+
+
+class SDKRelayEventSerializer(serializers.Serializer):
+    """Serializer for SDK relay events."""
+
+    token = serializers.RegexField(regex=r"^[0-9a-zA-Z]{32}$", required=True)
+    event = serializers.JSONField(required=True)
+
+    def validate_event(self, value):
+        """Validate that the event JSON data doesn't exceed maximum length."""
+
+        # Convert the JSON data to a string to check its length
+        json_string = json.dumps(value)
+        # One selected file could use up to 1000 characters, this limit is set to
+        # avoid DDOS attacks to fullfill the redis.
```

Comment:
I don't remember, is there a rate limit on this endpoint?

Reply:
Yes, there is a throttle https://github.com/suitenumerique/drive/issues/294 ^^